erwinhenraat/VoorbeeldExamenRepo
# Feature/seed input

# What Changed:
You can now put in a seed given to you and it will randomise the maze connected to the seed

# How to Test:
- Start the boot scene
- Copy this seed: 
   304klkakjaajaaa-023bdakbjjdaaaj-240410bdjegabajaa-431411eednkgnamaa-125410dtgmjaaaaea-kqnhbzsbdjama;3
- Paste the seed in the input field by clicking the paste button or your device way of pasting text
- Click the generate button and this maze will appear:
![image](https://github.com/erwinhenraat/VoorbeeldExamenRepo/assets/70703952/01d4064e-3648-419b-9d4c-dd809734c90c)

**Note:** this pull request shouldn't be merged with the current build settings
